pages/linux/xterm.md

```diff
@@ -0,0 +1,23 @@
+# xterm
+
+> The xterm command is a terminal emulator for the X Window System.
+
+- Open the terminal with a title of `Example`:
+
+`xterm -T {{Example}}`
+
+- Open the terminal in fullscreen mode:
+
+`xterm -fullscreen`
+
+- Open the terminal with a dark blue background and yellow font:
```

Comment:
Could you incorporate the term "foreground" in the description as well? That woul help make the connection with `bg` and `fg`.